From 9623bbedf4ff409e5036edfcfe52b2595932a6d7 Mon Sep 17 00:00:00 2001
From: Chris Larson <clarson@kergoth.com>
Date: Sat, 1 Nov 2008 20:46:07 +0000
Subject: [PATCH] Link plugins against libts

Some plugins use tslib functions. Link those plugins against libts.
The problem is easy to see with LDFLAGS="-Wl,-no-undefined".
Without this change DirectFB in unable to use tslib because symbols
in the tslib plugins can't be resolved.

Signed-off-by: Ville Syrjala <syrjala@sci.fi>
Signed-off-by: Chris Larson <clarson@kergoth.com>

The patch was imported from git server git://github.com/kergoth/tslib.git
as of commit id 9623bbedf4ff409e5036edfcfe52b2595932a6d7.

Upstream-Status: Accepted
Signed-off-by: Dmitry Cherukhin <dima_ch@emcraft.com>
---
 plugins/Makefile.am |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/plugins/Makefile.am b/plugins/Makefile.am
index 3b902c2..4c4ef8b 100644
--- a/plugins/Makefile.am
+++ b/plugins/Makefile.am
@@ -114,15 +114,19 @@ pluginexec_LTLIBRARIES = \
   
 variance_la_SOURCES	= variance.c
 variance_la_LDFLAGS	= -module $(LTVSN)
+variance_la_LIBADD	= $(top_builddir)/src/libts.la
   
 dejitter_la_SOURCES	= dejitter.c
 dejitter_la_LDFLAGS	= -module $(LTVSN)
+dejitter_la_LIBADD	= $(top_builddir)/src/libts.la
   
 linear_la_SOURCES	= linear.c
 linear_la_LDFLAGS	= -module $(LTVSN)
+linear_la_LIBADD	= $(top_builddir)/src/libts.la
 
 pthres_la_SOURCES	= pthres.c
 pthres_la_LDFLAGS	= -module $(LTVSN)
+pthres_la_LIBADD	= $(top_builddir)/src/libts.la
 
 # hw access
 corgi_la_SOURCES	= corgi-raw.c
@@ -148,6 +152,7 @@ tatung_la_LDFLAGS	= -module $(LTVSN)
 
 input_la_SOURCES	= input-raw.c
 input_la_LDFLAGS	= -module $(LTVSN)
+input_la_LIBADD		= $(top_builddir)/src/libts.la
 
 linear_h2200_la_SOURCES	= linear-h2200.c
 linear_h2200_la_LDFLAGS	= -module $(LTVSN)
-- 
1.7.6.4

